Formula/lib/libnice.rb

```diff
@@ -20,13 +20,14 @@ class Libnice < Formula
     sha256               x86_64_linux:   "392d7eb8bea2d013695a4d57d7a5849d702d018f9d1e2009accd01edbc979784"
   end
 
+  keg_only "it conflicts with gstreamer, which depends on and also includes libnice"
```

Comment:
I don't think making this keg-only is a good solution.

Reply:
reverted.